features/svg.yml

```diff
@@ -0,0 +1,901 @@
+  - svg.elements.feConvolveMatrix.bias
+  - svg.elements.feConvolveMatrix.divisor
+  - svg.elements.feConvolveMatrix.edgeMode
+  - svg.elements.feConvolveMatrix.in
+  - svg.elements.feConvolveMatrix.kernelMatrix
+  - svg.elements.feConvolveMatrix.kernelUnitLength
+  - svg.elements.feConvolveMatrix.order
+  - svg.elements.feConvolveMatrix.preserveAlpha
+  - svg.elements.feConvolveMatrix.targetX
+  - svg.elements.feConvolveMatrix.targetY
+  - svg.elements.feDiffuseLighting
+  - svg.elements.feDiffuseLighting.diffuseConstant
+  - svg.elements.feDiffuseLighting.in
+  - svg.elements.feDiffuseLighting.kernelUnitLength
+  - svg.elements.feDiffuseLighting.surfaceScale
+  - svg.elements.feDisplacementMap
+  - svg.elements.feDisplacementMap.in
+  - svg.elements.feDisplacementMap.in2
+  - svg.elements.feDisplacementMap.scale
+  - svg.elements.feDisplacementMap.xChannelSelector
+  - svg.elements.feDisplacementMap.yChannelSelector
+  - svg.elements.feDistantLight
+  - svg.elements.feDistantLight.azimuth
+  - svg.elements.feDistantLight.elevation
+  - svg.elements.feDropShadow
+  - svg.elements.feDropShadow.dx
+  - svg.elements.feDropShadow.dy
+  - svg.elements.feDropShadow.in
+  - svg.elements.feDropShadow.stdDeviation
+  - svg.elements.feFlood
+  - svg.elements.feFlood.flood-color
+  - svg.elements.feFlood.flood-opacity
+  - svg.elements.feFuncA
+  - svg.elements.feFuncB
+  - svg.elements.feFuncG
+  - svg.elements.feFuncR
+  - svg.elements.feGaussianBlur
+  - svg.elements.feGaussianBlur.in
+  - svg.elements.feGaussianBlur.stdDeviation
+  - svg.elements.feImage
+  - svg.elements.feImage.href
+  - svg.elements.feImage.preserveAspectRatio
+  - svg.elements.feMerge
+  - svg.elements.feMergeNode
+  - svg.elements.feMergeNode.in
+  - svg.elements.feMorphology
+  - svg.elements.feMorphology.in
+  - svg.elements.feMorphology.operator
+  - svg.elements.feMorphology.radius
+  - svg.elements.feOffset
+  - svg.elements.feOffset.dx
+  - svg.elements.feOffset.dy
+  - svg.elements.feOffset.in
+  - svg.elements.fePointLight
+  - svg.elements.fePointLight.x
+  - svg.elements.fePointLight.y
+  - svg.elements.fePointLight.z
+  - svg.elements.feSpecularLighting
+  - svg.elements.feSpecularLighting.in
+  - svg.elements.feSpecularLighting.specularConstant
+  - svg.elements.feSpecularLighting.specularExponent
+  - svg.elements.feSpecularLighting.surfaceScale
+  - svg.elements.feSpotLight
+  - svg.elements.feSpotLight.limitingConeAngle
+  - svg.elements.feSpotLight.pointsAtX
+  - svg.elements.feSpotLight.pointsAtY
+  - svg.elements.feSpotLight.pointsAtZ
+  - svg.elements.feSpotLight.specularExponent
+  - svg.elements.feSpotLight.x
+  - svg.elements.feSpotLight.y
+  - svg.elements.feSpotLight.z
+  - svg.elements.feTile
+  - svg.elements.feTile.in
+  - svg.elements.feTurbulence
+  - svg.elements.feTurbulence.baseFrequency
+  - svg.elements.feTurbulence.numOctaves
+  - svg.elements.feTurbulence.seed
+  - svg.elements.feTurbulence.stitchTiles
+  - svg.elements.feTurbulence.type
```

Comment:
Caniuse has a feature SVG filters https://caniuse.com/svg-filters (these lines plus the corresponding API stuff above).

It's a fairly old feature on caniuse (it was added no later than 2012), so if you think it's sufficiently historic to ignore for now that's fine, but I wanted to raise the possibility now, just in case.

Reply:
I like this idea, and seeing as it comprises 1/3rd of the keys, it seems like a reasonable split. e